index.html

```diff
@@ -1728,11 +1728,11 @@ <h2 id="AudioContext">
             </p>
           </dd>
           <dt>
-            readonly attribute AudioTimeStamp outputTimeStamp
+            AudioTimeStamp getOutputTimestamp()
           </dt>
           <dd>
             <p>
-              Holds an <a><code>AudioTimeStamp</code></a> instance containing
+              Returns an <a><code>AudioTimeStamp</code></a> instance containing
```

Comment:
"Return a new `AudioTimeStamp` object ..." for clarity? Also might want to rename to `AudioTimestamp` for consistency.